packages/fetchai/skills/oracle_aggregation/skill.yaml

```diff
@@ -0,0 +1,56 @@
+name: oracle_aggregation
+author: fetchai
+version: 0.1.0
+type: skill
+description: The skill for aggregating observations for oracle networks
+license: Apache-2.0
+aea_version: '>=0.9.0, <0.10.0'
+fingerprint:
+  __init__.py: QmPGxMBBZ8CRvLe3aiN8dTw62jj8PHWk3XJSRCqdk9TjBb
+  behaviours.py: QmSg3YDtQn1GijkVm3VcM7RJKh71XPuXJeiy86qkE6e4pa
+  dialogues.py: QmWwXSWWLgrw9Hwawym2g8CpmbS46detq2MWPrRU8eckne
+  handlers.py: QmcfYneKjLjQ6WZfab2md6TjnhHSe6XU4fDtbGq4mYqnpY
+  strategy.py: QmaihdgZ8QSNMiM7jHHFrMorxk9Dt1s2xf99RWHo1jKrUv
+fingerprint_ignore_patterns: []
+contracts: []
+protocols:
+- fetchai/default:0.11.0
+- fetchai/oef_search:0.12.0
+skills: []
+behaviours:
+  search:
+    args:
+      search_interval: 5
+    class_name: GenericSearchBehaviour
+handlers:
+  consensus:
+    args: {}
+    class_name: GenericConsensusHandler
+  oef_search:
+    args: {}
+    class_name: GenericOefSearchHandler
+models:
+  consensus_dialogues:
+    args: {}
+    class_name: ConsensusDialogues
+  default_dialogues:
+    args: {}
+    class_name: DefaultDialogues
+  oef_search_dialogues:
+    args: {}
+    class_name: OefSearchDialogues
+  strategy:
+    args:
+      location:
+        latitude: 51.5194
+        longitude: 0.127
+      search_query:
+        constraint_type: ==
+        search_key: generic_service
+        search_value: generic_service
```

Comment:
It's a good idea to be more specific. sOEF is increasingly busy. This might interfere with other registered agents.